sdk/dns/Azure.ResourceManager.Dns/src/Azure.ResourceManager.Dns.csproj

```diff
@@ -1,28 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>1.0.0-preview.2</Version>
+    <Version>1.0.0-beta.1</Version>
     <PackageId>Azure.ResourceManager.Dns</PackageId>
     <Description>
-      Azure management client SDK for Azure resource provider Microsoft.EventHub.
+      Azure management client SDK for Azure resource provider Microsoft.Dns.
       This is a beta preview vesion. This version uses a next-generation code generator that introduces important breaking changes, but also new features (such as intuitive authentication, custom HTTP pipeline, distributed tracing and much more).
     </Description>
     <Description>Azure Resource Manager client SDK for Azure resource provider Microsoft.Dns</Description>
     <PackageTags>azure;management;arm;resource manager;dns</PackageTags>
-    <TemporaryUsePreviousGeneratorVersion>false</TemporaryUsePreviousGeneratorVersion>
-    <FileUpgradeFlags>
-    </FileUpgradeFlags>
-    <UpgradeBackupLocation>
-    </UpgradeBackupLocation>
-    <OldToolsVersion>2.0</OldToolsVersion>
-    <IsMgmtSubLibrary>false</IsMgmtSubLibrary>
-  </PropertyGroup>
-  <PropertyGroup>
-    <!-- TODO: As part of release process, this PropertyGroup can be removed once RP is approved for namespace -->
     <NoWarn>$(NoWarn);CS0618</NoWarn>
```

Comment:
We actually do not have anything marked with `Obsolete`, we can remove this `NoWarn`
```suggestion
```